### ISSUE TITLE
Only two functions in fontTools.cu2qu.ufo has anything to do with UFO

### ISSUE THREAD

**Issue by simoncozens** [fonttools/fonttools#2184]
I'm trying to convert multiple outlines to interpolation-compatible quadratics; I was quite surprised to find that the function for this is `fontTools.cu2qu.ufo. glyphs_to_quadratic`, because it didn't occur to me to look in a library called `ufo`. There doesn't seem to be anything UFO-specific about this, as it operates on a list of any object with a `draw` method.

`font_to_quadratic` and `fonts_to_quadratic` are UFO-specific, but the rest of that library is pretty general. Can we move the functions elsewhere?

**Comment by anthrotype** [comment]
I think it's not just `draw` (like fonttools read-only _TTGlyph do) but also a `getPen` to allow writing back onto those glyph objects. I think this is only true for robofab-like APIs (defcon, fontParts, ufoLib2) so it made sense to keep that in the cu2qu.ufo sub-module.

> Can we move the functions elsewhere?

I'm open to suggestions.

**Comment by simoncozens** [comment]
> I think it's not just draw (like fonttools read-only _TTGlyph do) but also a getPen to allow writing back onto those glyph objects

You're right - `draw` and `getPen`. Just implemented it in nfsf.Layer, it works brilliantly!

>  I think this is only true for robofab-like APIs (defcon, fontParts, ufoLib2) so it made sense to keep that in the cu2qu.ufo sub-module.

Well, the interface is whatever we document the interface to be. (Currently we just say it's a glyph, which isn't helpful...) In one sense, yes, it's UFO-like, but it's also duck-typed to anything that supports `draw` and `getPen`.

> I'm open to suggestions.

How about we have a `fontTools.cu2qu.interpolatable` module, move the `glyphs_to_quadratic` functions and support logic there, document the expected interface, and keep `fontTools.cu2qu.ufo` for the `fonts_to_quadratic` functions and backwards compatibility?

**Comment by anthrotype** [comment]
Sounds good to me
